profile/usermanagement.php

```diff
@@ -61,20 +63,25 @@
 	<div id="innertext">
 		<div style="float:right;">
 			<div style="margin:10px 0px 15px 0px;">
-				<fieldset style="background-color:#FFFFCC;padding:0px 10px 10px 10px;">
-					<legend style="font-weight:bold;"><?php echo (isset($LANG['SEARCH'])?$LANG['SEARCH']:'Search'); ?></legend>
+				<section class="fieldset-like" style="background-color:#FFFFCC;">
+					<h1> <span> <?php echo (isset($LANG['SEARCH'])?$LANG['SEARCH']:'Search'); ?> </span> </h1>
 					<?php echo (isset($LANG['LAST_OR_LOGIN'])?$LANG['LAST_OR_LOGIN']:'Last Name or Login Name'); ?>:
 					<form name='searchform1' action='usermanagement.php' method='post'>
 						<input type="text" name="searchterm" title="<?php echo (isset($LANG['ENTER_LAST'])?$LANG['ENTER_LAST']:'Enter Last Name'); ?>" /><br/>
 						<button name="submit" type="submit" value="Search"><?php echo (isset($LANG['SEARCH'])?$LANG['SEARCH']:'Search'); ?></button>
 					</form>
+
+					<span style="text-indent: -5000px; position: absolute;">
```

Comment:
The class "skip-link" in css/v202209/symbiota/main.css should be used here; it is essentially the same thing.